### PR TITLE
Minor fixes.

### PR DESCRIPTION
**Pull request by armm77**
Fixed: Error saving the background color.
Fixed: Font has been updated to Liberation Mono (Terminus has been removed from the font).

### CODE REVIEW

File: Applications/Preferences/Modules/Display/Display.m [hunk 1]
```diff
@@ -42,7 +42,7 @@
 #import "AppController.h"
 #import "Display.h"
 
-NSString *NXTDesktopBackgroundColor = @"DesktopBackgroundColor";
+NSString *NXTDesktopBackgroundColor = @"NXDesktopBackgroundColor";
```
Comment (trunkmaster) [inline Applications/Preferences/Modules/Display/Display.m:45]: Have you noticed error on saving background color? Changing name just creates new key in NXGlobalDomain and should not lead to any error. I'd like to hold new name (without NX prefix) and change default values (and key) in /etc/skel.

Comment (armm77) [reply Applications/Preferences/Modules/Display/Display.m:45]: I downloaded version 0.90 for Centos 7, to do the example. I see that the changes are not saved. No problem you can close this change, with another more satisfactory solution.

example

Comment (trunkmaster) [reply Applications/Preferences/Modules/Display/Display.m:45]: @armm77 now I know where the problem resides (SystemKit uses hardcoded key). I'm going to merge this PR and move this definition into OSScreen class header file.

Comment (armm77) [reply Applications/Preferences/Modules/Display/Display.m:45]: @trunkmaster I made the rpms with the scripts, and the changes are not yet saved in plist.

**Comment by trunkmaster** [comment]
@armm77 Fixed in SystemKit: I missed another hardcoded key name string.

**Pull request merged**